keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,73 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    """Pads the given bounding boxes to convert it from a ragged tensor to
+    a regular tensor.
+
+    Args:
+      bounding_boxes: a ragged tensor of bounding boxes in 'corners' format.
```

Comment:
does it have to be in any specific format?

Reply:
yeah that is unnecessary. Removed it.